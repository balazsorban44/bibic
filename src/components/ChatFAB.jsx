import React from 'react'
import chat from '../assets/icons/chat.svg'
import Fade from "react-reveal/Fade"
import {useTranslation} from 'react-i18next'

export const ChatFAB = () => {
  const [t] = useTranslation()
  return (
    <Fade up>
      <div className="chat-fab">
        <span className="tooltip">{t("chat-fab")}</span>
        <a
          href="https://www.messenger.com/t/200199203718517"
          rel="noopener noreferrer"
          target="_blank"
        >
          <img
            alt=""
            aria-hidden
            className="filled"
            src={chat}
          />
        </a>
      </div>
    </Fade>
  )
}


export default ChatFAB