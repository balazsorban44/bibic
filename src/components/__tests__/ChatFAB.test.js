import {ChatFAB} from "../ChatFAB"
import Fade from "react-reveal/Fade"


describe("ChatFAB component", () => {
  const props = {hero: []}
  const wrapper = shallow(<ChatFAB {...props}/>)

  it("renders correctly", () => {
    expect(wrapper).toMatchSnapshot()
  })

  it("Fades when hero images loaded", () => {
    wrapper.setProps({hero: [1,2,3]})
    expect(wrapper.find(Fade).prop("when")).toBe(true)
  })
})