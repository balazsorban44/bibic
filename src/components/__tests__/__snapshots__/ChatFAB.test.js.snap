// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChatFAB component renders correctly 1`] = `ShallowWrapper {}`;
